flexmeasures/ui/utils/breadcrumb_utils.py

```diff
@@ -73,7 +74,11 @@ def get_siblings(entity: Sensor | Asset | Account | None) -> list[dict]:
         if entity.parent_asset is not None:
             sibling_assets = entity.parent_asset.child_assets
         else:
```

Comment:
We want to keep support for non-public top-level assets, though.

```suggestion
        elif entity.parent_asset is not None:
            sibling_assets = entity.owner.generic_assets
        else:
```